cilium/cmd/policy_trace.go

```diff
@@ -256,22 +256,21 @@ func getSecIDFromK8s(podName string) (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("unable to create rest configuration: %s", err)
 	}
-	k8sClient, err := k8s.CreateClient(restConfig)
+	ciliumK8sClient, err := clientset.NewForConfig(restConfig)
 	if err != nil {
 		return "", fmt.Errorf("unable to create k8s client: %s", err)
 	}
 
-	p, err := k8sClient.CoreV1().Pods(namespace).Get(pod, meta_v1.GetOptions{})
+	ep, err := ciliumK8sClient.CiliumV2().CiliumEndpoints(namespace).Get(pod, meta_v1.GetOptions{})
```

Comment:
What do you think about providing a warning that we need the CiliumEndpoint CRD enabled in case the pod with the identity cannot be found? Maybe documenting in the help info for this command would suffice.

Reply:
done